remoteit/desktop
# DESK-864 allow copying of launch and copy commands

### Description


Allow copying of launch and copy commands
Show your own cloud connections

### Test plan



1. Sign in
2. Go to a service 
3. Edit the launch or copy command with an empty token
4. Click the clipboard to the left of the command
5. If the token if empty you should be prompted to enter a value
6. The command should be copied to your clipboard

### Ticket



<https://remoteit.atlassian.net/browse/DESK-864>
